android/expoview/src/main/java/host/exp/exponent/ReactNativeStaticHelpers.kt

```diff
@@ -0,0 +1,184 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+package host.exp.exponent
+
+import android.util.Log
+import com.facebook.proguard.annotations.DoNotStrip
+import com.facebook.react.common.JavascriptException
+import host.exp.exponent.network.ExponentNetwork
+import host.exp.expoview.Exponent
+import okhttp3.CookieJar
+import okhttp3.OkHttpClient
+import java.util.concurrent.TimeUnit
+
+@DoNotStrip
+object ReactNativeStaticHelpers {
+  private val TAG = ReactNativeStaticHelpers::class.java.simpleName
+
+  private var bundleUrl: String? = null
+  @DoNotStrip
```

Comment:
Not sure we need this (or if it is even called for that matter...)

Reply:
It isn't called, but I'll remove in a follow-up since these dynamically-called static methods are probably the most fragile part of the client (no compiler help). https://linear.app/expo/issue/ENG-1831/remove-reactnativestatichelpersbundleurl